build.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
```

Comment:
Could you revert this change? I would like to avoid adding too much information so that folks can focus on the essential parts. This is already explained in the README.